app/styles/modules/_global_header.scss

```diff
@@ -2,7 +2,7 @@
   @include color-scheme($white, $middle-grey);
   position: fixed;
   top: 0;
-  z-index: 100;
+  z-index: 10;
```

Comment:
fine for now, but we might want to be careful about layering the z-indexes, this might be a good use case for variables, so we can see the full list of z-indexes in one place.